modules/tinymce/src/plugins/image/test/ts/browser/plugin/MainTabTest.ts

```diff
@@ -47,4 +48,29 @@ describe('browser.tinymce.plugins.image.plugin.MainTabTest', () => {
       '</figure>'
     ));
   });
+
+  it('TINY-6400: render image_class_list as Dialog type "panel"', async () => {
+    const editor = hook.editor();
+    editor.settings.image_caption = false;
+    editor.execCommand('mceImage');
+
+    const dialog = await TinyUiActions.pWaitForDialog(editor);
+    const expected = {
+      ['.tox-form__group button[aria-label="Class"]']: 1,
+      ['.tox-form__grid--2col button[aria-label="Class"]']: 0
+    };
+    Assertions.assertPresence('Asserting presence', expected, dialog);
```

Comment:
```suggestion
    Assertions.assertPresence('Does not have columns for the class', expected, dialog);
```
This really should say what it's asserting the presence of instead of just saying it's asserting something without any context.

Reply:
updated